### ISSUE TITLE
allow multi-repack jobs to be resumed

### ISSUE THREAD

**Issue by Masamune3210** [julesontheroad/NSC_BUILDER#214]
currently if the program gets closed or something else happens to make the program stop, the operation must be restarted completely, unlike the rest of the operations like repacking as nsp where if you close the program you can resume the job once you reopen the program

perhaps allow multi-content repacks to operate under the same idea? the lists currently get deleted the moment the multi-pack menu is opened after the program is restarted

**Comment by julesontheroad** [comment]
I think is because is made on the assumption that on the list there's only content that will be packed into a file.
I think that you can do that if you save the lists to process later and then you process saved jobs since jobs aren't deleted till completed.

**Comment by julesontheroad** [comment]
You can also use the function that separates content by baseid and save the jobs, that allows you to add different files indiscriminately to the tool and it will decide which goes in which list.

**Comment by Masamune3210** [comment]
I use the separate by base id, but if I have to stop the job or something happens and the program closes, it doesn't offer to resume like the other options like verifying files. It just immediately deletes the list and acts like they weren't even there. I'm probably failing to explain it well, let me know if so

**Comment by julesontheroad** [comment]
Yeah I know what you mean what I mean is that you need to use the option to save lists to process later and then process previously saves lists. The mlist.txt itself was designed to not be saved that's why there's the option to save the lists to process them later when one is processed is deleted and the queue is generated when you use the option to process previously saves lists.

**Comment by Masamune3210** [comment]
Ok, I see the option now, I guess I looked it over and didnt notice it since the other actions work differently. Thanks!

**Comment by Masamune3210** [comment]
Got the feature to work, could files be output into the output folder instead of the temp folder so they arent automatically deleted if the program is relaunched and for completed items to be deleted from the list so they arent reprocessed next time?

**Comment by Masamune3210** [comment]
Ignore the remove from list part as I apparently didnt notice it did that already, the placement of processed files in temp is dangerous though as if the files arent moved before the program is restarted, the files will be removed when the program cleans up the temp dir
